readme.az.common.md

```diff
@@ -1,7 +1,6 @@
 # configuration for az common
  
 ``` yaml $(az)
```

Comment:
suggest updating the document for the default module mode.

Reply:
okay.

Reply:
updated in here https://github.com/Azure/autorest.az/pull/797/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R79-R80